miniapps/nurbs/nurbs_ex24.cpp

```diff
@@ -0,0 +1,461 @@
+//                                MFEM Example 24 -- modified for NURBS FE
+//
+// Compile with: make nurbs_ex24
+//
+// Sample runs:  nurbs_ex24 -m ../../data/pipe-nurbs-2d.mesh -o 2
+//               nurbs_ex24 -m ../../data/pipe-nurbs-2d.mesh -p 2
+//               nurbs_ex24 -m ../../data/cube-nurbs.mesh -o 2
+//               nurbs_ex24 -m ../../data/cube-nurbs.mesh -o 2 -p 1
+//               nurbs_ex24 -m ../../data/cube-nurbs.mesh -o 2 -p 2
+//               nurbs_ex24 -m ../../data/escher.mesh
+//               nurbs_ex24 -m ../../data/escher.mesh -o 2
+//               nurbs_ex24 -m ../../data/fichera.mesh
+//               nurbs_ex24 -m ../../data/fichera-q2.vtk
+//               nurbs_ex24 -m ../../data/fichera-q3.mesh
+//               nurbs_ex24 -m ../../data/square-disc-nurbs.mesh
```

Comment:
`square-disc-nurbs.mesh` doesn't work, but `square-nurbs.mesh` does. I guess these sample runs were copied and pasted, and we should have fewer, more basic runs with `square-nurbs` and `cube-nurbs`.

Reply:
This is most likely a multi-patch mesh, which currently does not work.
I thought I removed those.

Reply:
`square-disc-nurbs.mesh` and `beam-hex-nurbs.mesh` are multi patch. I removed those examples.